datafusion/core/src/execution/context.rs

```diff
@@ -1489,6 +1381,15 @@ impl SessionConfig {
     }
 }
 
+impl From<ConfigOptions> for SessionConfig {
```

Comment:
This lays the ground-word for eventually removing SessionConfig altogether #3887